L.Control.Geonames.js

```diff
@@ -5,7 +5,7 @@ var BBOX = ['east', 'west', 'north', 'south'];
 L.Control.Geonames = L.Control.extend({
     includes: L.Mixin.Events,
 
-    _active: false,
+    _active: true,
```

Comment:
Instead of setting here, we want to make this an option that is passed in to the constructor.

Suggestion: `alwaysOpen: false` which will mimic current behavior for this widget in current use, and then just set it to `true` when you construct in mobile view.